metricbeat/helper/http_test.go

```diff
@@ -236,14 +237,14 @@ func TestOverUnixSocket(t *testing.T) {
 			fmt.Fprintf(w, "ehlo!")
 		})
 
-		go http.Serve(l, mux)
+		go http.Serve(l, mux) //nolint:all // Ignore the error
```

Comment:
Whereas muting a linter is fine, we aim to only mute necessary linter and add a comment explaining why the linter is disabled. Juts muting all linter to ignore their errors isn't ideal